source/elements/oneDAL/source/terminology.rst

```diff
@@ -0,0 +1,75 @@
+.. _terminology:
```

Comment:
This is what I see in the https://github.com/napetrov/oneapi-spec/blob/dev/napetrov-introduction/source/elements/oneDAL/source/terminology.rst

Formatting issues



Term | Definition
-- | --
Batch Mode | The computation mode for an algorithm in oneDAL, where all the data needed for

omputation are available at the start and fits the memory of the device where computations will be performed.

Dataset
A pair of a matrix $X$ of size $N times p$ and a vector $Y$ of size $N$, where $X$
contains $N$ feature vectors in rows and $Y$ stores all corresponding responses. Vector $Y$ is optional and may not be present if responses are unknown.

Not all comments were addressed

Reply:
terminology formatting issues dressed

![image](https://user-images.githubusercontent.com/18015221/77197380-209ed280-6af6-11ea-993c-bf1a2ff38096.png)